src/Console/Command/Compile.php

```diff
@@ -852,4 +867,30 @@ private function logEndBuilding(CompileLogger $logger, SymfonyStyle $io, Box $bo
             )
         );
     }
+
+    private function renderMessages(Configuration $config, SymfonyStyle $io): void
+    {
+        $recommendations = $config->getRecommendations();
+        $warnings = $config->getWarnings();
+
+        if ([] === $recommendations) {
+            $io->writeln('No recommendation found.');
+        } else {
+            $io->writeln('Recommendations:');
+
+            foreach ($recommendations as $recommendation) {
```

Comment:
It's not strictly important, but I tend to lean towards using an `array_walk` these days, to take advantage of typing, e.g.:

```php
array_walk(
  $recommendations
  function (string $recommendation): void {
    $io->writeln("    - <recommendation>$recommendation</recommendation>");
  }
);
```

I know it doesn't add *much* but just thought I'd throw this into the mix :D